server/services/v1/realtime/device_session.go

```diff
@@ -146,6 +147,7 @@ func (session *Session) Start(ctx context.Context) error {
 		}
 		session.lastReceived = time.Now()
 		if errEvent := session.onMessage(ctx, message); errEvent != nil {
+			log.Info().Msgf("sending error", errEvent)
```

Comment:
How verbose can this be? Should it be a metric instead?

Reply:
I think we will add metrics in a later PR

Reply:
We can log it as an error, this should be rare.